### PR TITLE
Command clearing

### PR DESCRIPTION
**Pull request by rmevans9**
I have actually had this laying around for a while and just now realized it was not ever PR'd. So here it is.

### CODE REVIEW

File: packages/reactotron-server/src/server/datastore/commands.ts [hunk 1]
```diff
@@ -54,7 +54,20 @@ export class Commands {
   commands: Command[] = []
 
   addCommand(command: Command) {
-    this.commands.push(command)
+    this.commands.unshift(command)
+
+    if (this.commands.length > 300) { // Arbitrary, I know... but its a start somewhere
+      this.commands = this.commands.slice(0, 300)
```
Comment (ianoshorty) [inline packages/reactotron-server/src/server/datastore/commands.ts]: Sometimes magic numbers are required, but if they are, I usually like to define them in a named constant so their purpose is clear. Its not obvious from this line why or what 300 is / does

Comment (skellock) [reply packages/reactotron-server/src/server/datastore/commands.ts]: I agree. I’d like this number to be configurable too. So maybe a constructor to pass it in? doesnt have to happen now though.

File: packages/reactotron-server/src/server/reactotron/commandHandler.ts [hunk 2]
```diff
@@ -12,6 +12,12 @@ import { pluginManager } from "../pluginManager"
  * @param command A command sent from the client.
  */
 export function onCommand(command: Command) {
+  // TODO: Consider a special place for these "system" level commands. Do we want them to propagate like a normal command?
```
Comment (skellock) [inline packages/reactotron-server/src/server/reactotron/commandHandler.ts]: excellent point

File: packages/reactotron-server/src/server/datastore/commands.ts (hunk 1 above)
Comment (rmevans9) [reply packages/reactotron-server/src/server/datastore/commands.ts]: Constructor pass in setup but didn't set up anything to actually pass it in. That part can come later.

**Pull request merged**